operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -352,4 +353,27 @@ public void testInitialAdminUser() {
         }
     }
 
+    @Test
+    @EnabledIfSystemProperty(named = OPERATOR_CUSTOM_IMAGE, matches = ".+")
```

Comment:
Maybe, I'd just use something like `assumeNotNull(customImage)` not to explicitly rely on system property as the `customImage` can be set not only by system property.

Reply:
in `junit 5` we have `assumingThat`, but I don't really like the outcome:

   - using the annotation the skipped test is evident in the tests report
   - using `assumingThat` the test is marked as green even if it wasn't executed
   
this is really confusing for me. I believe that if we are really going down the path to skip tests by default it should be evident what is happening.

Do you see us setting this property in any other way? Do you know any alternative option?

Reply:
as agreed we leave this with the Annotation, at least for now.